examples/soak/setup_ecs_cluster.yml

```diff
@@ -0,0 +1,242 @@
+AWSTemplateFormatVersion: '2010-09-09'
```

Comment:
there're a few resources in here that'd be per-application, and a few that are per-cluster - e.g. the load balancer isn't application-specific (the forwarding rule is, though) - let's figure out what'll change together, and that's likely how best to split the CF up too